style.css

```diff
@@ -96,3 +96,271 @@ nav #mylogo {
+}
+
+.text-title {
+  display: flex;
+  flex-direction: row;
+  align-self: flex-start;
+  padding: 0;
+  gap: 12px;
+  align-items: center;
+}
+
+.client {
+  height: 14px;
+  font-style: normal;
+  font-weight: 600;
+  text-decoration: none;
+  font-size: 15px;
+  line-height: 16px;
+  margin: 0;
+  color: #344563;
+}
+
+.client-skills {
+  font-weight: 600;
+  font-size: 14px;
+  margin: 0;
+  line-height: 17px;
+  font-style: normal;
+  text-decoration: none;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  color: #7a869a;
+}
+
+.dot {
+  height: 8px;
+  width: 8px;
+  border-radius: 50%;
+  background-color: #c1c7d0;
+  display: inline-block;
+}
+
+.period {
+  font-size: 14px;
+  font-weight: 600;
+  margin: 0;
+  line-height: 16px;
+  font-style: normal;
+  text-decoration: none;
+  color: #7a869a;
+}
+
+.content-text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  color: #344563;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  padding: 0;
+  width: 203px;
+  height: 25px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  list-style: none;
+}
+
+.style {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  text-align: center;
+  line-height: 15px;
+  letter-spacing: 0.03em;
+  padding: 5px 12px;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+}
+
+.link {
+  padding: 13px;
+  gap: 10px;
+  left: 0;
+  border: 1px solid #6070ff;
+  border-radius: 9px;
+  letter-spacing: 0.04em;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 26px;
+  text-decoration: none;
+  text-align: center;
+  cursor: pointer;
+  order: 3;
+  margin: 12px 0;
+  color: #396df2;
+}
+
+.About-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  padding: 115px 25px;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 14px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 25px;
+  color: #344563;
+}
+
+.about-head {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 50px;
+  color: #091e42;
+}
+
+.about-skills {
+  padding: 0;
+  width: 327px;
+  height: 464.15px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.lang-detail {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.lang-head {
+  display: flex;
+  height: 24px;
+  align-items: center;
+  justify-content: space-between;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  font-style: normal;
+}
+
+.lang-names {
+  padding: 12px;
+  gap: 12px;
+  width: 327px;
+  height: 248.15px;
+}
+
+.arrow {
+  border: solid #acb7c3;
+  border-width: 0 3px 3px 0;
+  display: block;
+  padding: 5px;
+  margin-right: 10px;
+}
```

Comment:
- [ ] Make sure the arrow icon are like the one used in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=cXIiLGyoO0c48vzg-0). You can always export icons and images from [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=OfmWBrmzYCEUWnT6-0) if you don’t find similar ones online

**_Your design_**

<img width="225" alt="Screenshot 2023-02-17 at 11 16 35 PM" src="https://user-images.githubusercontent.com/44798044/219761277-fa596aaf-4358-4bb1-878e-78c4870b2f8f.png">

_**Figma design**_

<img width="385" alt="Screenshot 2023-02-17 at 11 21 09 PM" src="https://user-images.githubusercontent.com/44798044/219761527-49b6f65c-ece1-4221-9022-62001ae3c7b6.png">